applications/RANSApplication/python_scripts/formulations/turbulence_models/scalar_turbulence_model_rans_formulation.py

```diff
@@ -0,0 +1,208 @@
+from abc import abstractmethod
+
+# import kratos
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# import formulation interface
+from KratosMultiphysics.RANSApplication.formulations.rans_formulation import RansFormulation
+
+# import utilities
+from KratosMultiphysics.RANSApplication.formulations.utilities import CreateRansFormulationModelPart
+from KratosMultiphysics.RANSApplication.formulations.utilities import CreateBlockBuilderAndSolver
+from KratosMultiphysics.RANSApplication.formulations.utilities import InitializePeriodicConditions
+from KratosMultiphysics.RANSApplication.formulations.utilities import GetBoundaryFlags
+from KratosMultiphysics.RANSApplication.formulations.utilities import GetKratosObjectType
+from KratosMultiphysics.RANSApplication.formulations.utilities import CalculateNormalsOnConditions
+from KratosMultiphysics.RANSApplication.formulations.utilities import InitializeYPlusVariablesInConditions
+
+class ScalarTurbulenceModelRansFormulation(RansFormulation):
+    def __init__(self, model_part, settings):
+        """Scalar turbulence model formulation base class
+
+        This solves the variable given in self.GetSolvingVariable(), using element and conditions
+        having prefixes provided by self.GetElementNamePrefix() and self.GetConditionNamePrefix()
+
+        If wall functions are used, then self.GetConditionNamePrefix() should return non-empty prefix
+        otherwise it should be empty.
+
+        Args:
+            model_part (Kratos.ModelPart): ModelPart to be used in the formulation.
+            settings (Kratos.Parameters): Settings to be used in the formulation.
+        """
+        super().__init__(model_part, settings)
+
+        defaults = Kratos.Parameters(r"""{
+            "relative_tolerance"    : 1e-3,
+            "absolute_tolerance"    : 1e-5,
+            "max_iterations"        : 200,
+            "relaxation_factor"     : 0.5,
+            "echo_level"            : 2,
+            "linear_solver_settings": {
+                "solver_type"  : "amgcl"
+            },
+            "boundary_flags": ["INLET", "STRUCTURE"]
+        }""")
+
+        settings = self.GetParameters()
```

Comment:
Isn't `settings` a parameters object?

Reply:
yes you are correct, i changed the order... simple copy paste mistakte :|